eth2/beacon/fork_choice/lmd_ghost.py

```diff
@@ -21,6 +24,47 @@
 from eth2.beacon.typing import Epoch, Gwei, SigningRoot, Slot, Timestamp, ValidatorIndex
 from eth2.configs import CommitteeConfig, Eth2Config
 
+LMD_GHOST_SCORE_DATA_LENGTH = 2
+
+
+def score_block_by_root(root: SigningRoot) -> int:
+    return int.from_bytes(root, byteorder="big")
+
+
+class LMDGHOSTScore(BaseScore):
+    _score: Tuple[Gwei, int]
```

Comment:
Perhaps add a comment like:
```suggestion
    # First score by the latest attesting balance, and then score by block root
    _score: Tuple[Gwei, int]  
```